app/controllers/sessions_controller.rb

```diff
@@ -3,8 +3,6 @@
 class SessionsController < ApplicationController
   before_action :set_session, except: %i[index create]
 
-  layout "full", only: :edit
```

Comment:
So if I understand the SO correctly (or said another way), this makes it so that the `edit` action gets the `full` layout, but every other action gets reverted back to the `application` layout, instead of `two_thirds` as one might think because that's the default set in `ApplicationController`. Is that right?

Reply:
Yes, that's right. The `layout` call in `ApplicationController` isn't really a "default" as such, any call to `layout` in a controller that inherits from it completely replaces what's in `ApplicationController`.